src/build/__main__.py

```diff
@@ -33,13 +33,8 @@
     'underline': '\33[4m',
     'reset': '\33[0m',
 }
-
-
-def _print(message: str) -> None:
-    color = sys.stdout.isatty() and 'NO_COLOR' not in os.environ
-    for name, code in _STYLES.items():
-        message = message.replace(f'[{name}]', code if color else '')
-    print(message)
+if not sys.stdout.isatty() or 'NO_COLOR' in os.environ:
```

Comment:
I like that we have an environment variable to disable color, however can we have also one that force enables it? (e.g. tox 4 cannot forward the tty state into a subprocess yet, so is handy in such cases to force color mode). Perhaps FORCE_COLOR? And if neither set then check the tty state :blush:

Reply:
Sure. What happens with both `FORCE_COLOR` and `NO_COLOR`?

Reply:
In my implementation NO_COLOR wins, but :shrug:

Reply:
See e.g. how this would be used https://github.com/tox-dev/tox/blob/rewrite/tox.ini#L79 (this shows how mypy handles it).

Reply:
Done.

Reply:
Shouldn't empty values translate as false for these flags?

Reply:
https://no-color.org/ says the mere presence enables the behavior, the value is irrelevant. I think we should have the same behavior in `FORCE_COLOR`.

Reply:
I'm a fan of having a FORCE_COLOR too, as often on GHA (which supports color) things can be missing color. I think no-color.org went overboard - it's a pretty well established practice for an empty environment variable to be the same as unsetting it. It's not always easy to unset an environment variable, and the disconnect there is irritating. But I'd follow what the spec dictates. :(

Reply:
I understand the concern and share a similar opinion. IMO, coloring should be if `NO_COLOR` is present and it not `no`/`false`. But given that it is already a pretty well established standard, I think the best option is to follow what is defined.